tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+
+The command should output something similar to:
+
+
+    *** Recording 3 seconds with USB device 0 ***
+    *** Finished recording. Wrote output: test1.wav ***
+
+Note - you may get ALSA errors but you can ignore them, if the recording was
+successful.
+
+2. Listen to the test recording to make sure it worked and that the dialog
+sound quality is ok. Execute:
+
+        aplay test1.wav
+
+You may need to configure which interface the Pi uses for sound playback
+output. You can choose between the HDMI output and the 3.5mm line out /
+headphone jack, by executing: `sudo raspi-config`, and configuring the setting
+under `7: Advanced Options → A4 Audio → Choose the audio output (HDMI |
+Headphones)`.
+
+
+### Test the Media Translation API example Python client
+
+Now that your USB microphone works with Python, we can try to call the Media
+Translation API. This step will test calling the API in streaming mode, piping
+microphone audio to the service and displaying the translated live responses in
+ the command line shell.
+
+
+1. Execute the utility with the following command. You can specify the [target language code](https://cloud.google.com/translate/media/docs/languages). The default target language is `de-DE` for German. For example to test with Italian, execute the below command.
+In this example we use device number `0`. Replace that with your device number if
+necessary:
+
+        python3 translate-microphone.py --lang it-IT --dev 0
+
+Note: some of the target languages require a
+[Unicode](https://home.unicode.org/) font to be displayed correctly. By default
+the Raspberry Pi console font cannot display Unicode characters. For this
+reason, use a Latin-based language such as German or Italian in this step, to
+test the Media Translation API with your microphone. The next chapters will
+show how to use the `ar-subs.py` app which uses the pygame library and
+specific fonts, to correctly display output in for example Hindi and Japanese.
+
+
+### Test Pygame with an HDMI display
+
+Now that the microphone is ready and the Media Translation API works,  make sure that pygame can control the connected HDMI display or projector. We can test this by running an example game included in the pygame package.
+
+1. If you have been connected with SSH, you must now switch to the Pi's HDMI
+output console. I.e a monitor or a projector connected to the Pi's HDMI port
+and log in there.
+
+1. Change to the solution app directory:
+
+        cd community/tutorials/ar-subs
+
+1. Activate the python virtual environment:
+
+        source venv/bin/activate
+
+1. Execute the pygame test game with:
+
+        python3 -m pygame.examples.aliens
+
+1. You should see a game on the console display:
+
+![Pygame image][pygame]
+
+[pygame]: img/aliens.png
+
+6. You can control your car with the arrow keys and shoot with the space bar.
+7. The game will exit when you are hit.
+8. If you can see the game, pygame is working with your display and we can
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
Done